RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -189,12 +189,13 @@ extension NonFungibleTokenDetails.ViewState.TokenDetails {
 }
 
 extension OnLedgerEntity.NonFungibleToken.NFTData {
+	private static let standardFields: [OnLedgerEntity.NonFungibleToken.NFTData.StandardField] = [.name, .description, .keyImageURL]
```

Comment:
This is used for now to show the `claim_epoch` and `claim_amount` as arbitrary fields. It needs to be clarified if we should display those two fields in a different way in NFT details.

Reply:
This is a good idea, but for most, if not all, standard fields we will want to map the key to a localised string key, right?

Reply:
Standard fields do have a predefined way of being displayed - like the `name` is shown as the sheet title, the `keyImageURL` is shown as an image. We don't actually display the name of the field, we do show its content in some predefined way.
But true, if there is ever the case of showing the name of the standard field, it would be localised.

Reply:
Sure, for those ones, but this PR makes me think that there might be fields that should be displayed as labeled values. I suppose actually it would make more sense to separate that concept from the standard field concept, and just have something like `fieldNameMapping: [OnLedgerEntity.NonFungibleToken.NFTData.AnyKey: LocalizedStringKey]`

Reply:
ArbitraryFields are the ones that are named, but not localised. It was introduced this with my previous PR.